keyform/migrations/0010_auto_20170811_1848.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-08-11 18:48
+from __future__ import unicode_literals
+
+from decimal import Decimal
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('keyform', '0009_auto_20170804_1840'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='keydata',
+            name='core_number',
+            field=models.CharField(blank=True, help_text='Use a comma to separate the list of cores (if suite style).', max_length=35, verbose_name='New Core Number'),
+        ),
+        migrations.AlterField(
+            model_name='keydata',
+            name='quantity',
+            field=models.IntegerField(help_text="If you don't need to order more keys because there are enough already, you can mark zero.", validators=[django.core.validators.MinValueValidator(0)]),
+        ),
+        migrations.AlterField(
+            model_name='request',
+            name='amt_received',
+            field=models.DecimalField(blank=True, decimal_places=2, default=0, help_text='Core Change/Reprogram Fob Charge = $50, Room Key/Fob = $10, Mailbox Key = $10, Monroe Mailbox Key = $25', max_digits=7, validators=[django.core.validators.MinValueValidator(Decimal('0.00'))], verbose_name='Amount received'),
+        ),
+        migrations.AlterField(
+            model_name='request',
+            name='payment_method',
+            field=models.CharField(choices=[('ca', 'Cash'), ('ch', 'Check'), ('na', 'Not Applicable')], help_text="Cash/Check should only be accepted during camps and conferences, and also fill in the amount received. Use the Charge Amount box to change to the student's account, or mark that the student was charged on the RCR if they are checking out.", max_length=2),
```

Comment:
You'll need to create a new migration since this help_text is no longer part of models.py